Data/Sys/GameSettings/SD2J01.ini

```diff
@@ -26,8 +26,8 @@ PH_ZNear =
 PH_ZFar =
 
 [Video_Settings]
-SafeTextureCacheColorSamples = 512
```

Comment:
Could you test what the performance difference is at your side? If it is important i wouldn't object for something so minor, but if it is minor as i suspect i think it is not worth it to remove it. Usually i got 1-2% difference at worst here with stc in the middle.